### PR TITLE
Announced speakers 3 & 4, Changes in Speakers page design

### PR DESCRIPTION
**Pull request by YJDave**
image

- [X]  Adding facebook option for speakers

- [X]  Adding designation in speaker profile, currently it doesn't show

- [X]  Little bit of styling to make all cards same size irrespective of their content size

This is the last commit, here is the new updated version:
image

### CODE REVIEW

**Comment by JaldeepAsodariya** [comment]
> 6d1c7ad : Add facebook profile link of speakers.

Please use **3B5998** hexa color for facebook icon 

> e39648b : Add speaker designation.

Remove **font-italic** attribute

**Comment by YJDave** [comment]
@JaldeepAsodariya updated!
image

File: src/views/speakers.vue [hunk 1]
```diff
@@ -52,6 +52,7 @@
                             </v-avatar>
                             <p class="mt-3 mb-0 google-font" style="font-size:120%">{{item.name}}</p>
                             <p class="mt-1 mb-0 google-font">{{item.designation}}</p>
+                            <v-divider></v-divider>
```
Comment (JaldeepAsodariya) [inline src/views/speakers.vue]: Please remove this divider.

Comment (YJDave) [reply src/views/speakers.vue]: Removed.

**Comment by JaldeepAsodariya** [comment]
@YJDave Review this below design (just for your reference)
http://devfest.gdgahmedabad.com/speakers/

**Comment by YJDave** [comment]
@JaldeepAsodariya new design
image

**Comment by JaldeepAsodariya** [comment]
@YJDave Any specific use of my.txt? If yes then please give it proper name or else remove it.

**Comment by YJDave** [comment]
It is not required, Removed in #9

**Pull request merged**